packages/manager/modules/pci/src/projects/project/billing/billing.routing.js

```diff
@@ -37,6 +38,18 @@ export default /* @ngInject */ ($stateProvider) => {
     resolve: {
       breadcrumb: /* @ngInject */ ($translate) =>
         $translate.instant('cpbc_billing_control'),
+      currentActiveLink: /* @ngInject */ ($transition$, $state) => () =>
+        $state.href($state.current.name, $transition$.params()),
+      billingLink: /* @ngInject */ ($state, projectId) =>
+        $state.href('pci.projects.project.billing', { projectId }),
+      estimateLink: /* @ngInject */ ($state, projectId) =>
+        $state.href('pci.projects.project.billing.estimate', { projectId }),
+      historyLink: /* @ngInject */ ($state, projectId) =>
+        $state.href('pci.projects.project.billing.history', {
+          projectId,
+          year: moment.utc().year(),
+          month: moment.utc().month() + 1,
```

Comment:
what if month === 12 ? maybe moment.add(1, 'month') ?

Reply:
nevermind, i though you were trying to add 1 month, but you are simply converting from 0-11 to 1-12